From 9e913b5ce0d1ab22d20bbae80fbb53960fe07161 Mon Sep 17 00:00:00 2001
From: Niklas Haas <git@haasn.xyz>
Date: Tue, 10 Oct 2017 03:45:38 +0200
Subject: [PATCH 17/22] wscript: switch to pkg-config for vulkan

Turns out upstream got their shit together and wrote a (broken)
vulkan.pc.

Is there literally a single person in the entire vulkan ecosystem who
understands how build systems and libraries work?
---
 wscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wscript b/wscript
index e25a7667ad..bb494c2f5c 100644
--- a/wscript
+++ b/wscript
@@ -831,7 +831,7 @@ video_output_features = [
     }, {
         'name': '--vulkan',
         'desc':  'Vulkan context support',
-        'func': check_cc(header_name='vulkan/vulkan.h', lib='vulkan'),
+        'func': check_pkg_config('vulkan', '>= 1.0'),
     }, {
         'name': '--shaderc',
         'desc': 'libshaderc SPIR-V compiler',
-- 
2.14.3

